src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/repository/PersonRepository.kt

```diff
@@ -22,4 +23,13 @@ interface PersonRepository : JpaRepository<PersonEntity, Long> {
       "WHERE offender.pncNumber = :pncNumber OR defendant.pncNumber = :pncNumber OR prisioner.pncNumber = :pncNumber",
   )
   fun findPersonEntityByPncNumber(@Param("pncNumber") pncNumber: PNCIdentifier?): List<PersonEntity>
+
+  @Query(
+    "SELECT distinct p FROM PersonEntity p " +
+      "LEFT JOIN DefendantEntity defendant on defendant.person.id = p.id " +
+      "LEFT JOIN OffenderEntity offender on offender.person.id = p.id " +
+      "LEFT JOIN PrisonerEntity prisioner ON prisioner.person.id = p.id " +
+      "WHERE offender.dateOfBirth = :dateOfBirth OR defendant.dateOfBirth = :dateOfBirth OR prisioner.dateOfBirth = :dateOfBirth",
```

Comment:
should be `prisoner` rather than `prisioner`

But surely we are not going to do this across >1m records?